### DISCUSSION TITLE
Where are the passwords stored

### DISCUSSION THREAD

**Discussion by Daszbin** [cefsharp/CefSharp#4299] [Ideas]
I’m curious if in future versions of Cefsharp if we could get sql data file where all the passwords are saved similar to Login Data on Chrome.

**Comment by amaitland** [comment]
In future please just ask your question once and wait for a reply before creating an issue. 

The CEF alloy runtime doesn't support password saving. This is an upstream limitation.

A quick search yields https://www.magpcss.org/ceforum/viewtopic.php?f=6&t=18608 which  suggests the CEF chrome runtime might support password saving. 

See https://github.com/cefsharp/CefSharp/discussions/4123 for details on the Chrome runtime. At this point in time I'd only recommend advanced users use the Chrome runtime.
